src/etc/inc/openvpn-client-export.inc

```diff
@@ -95,7 +95,7 @@ function openvpn_client_export_validate_config($srvid, $usrid, $crtid) {
 	{
 		$input_errors[] = gettext("Could not locate server certificate.");
 	} else {
-		$server_ca = isset($server_cert['caref']) ? lookup_ca($server_cert['caref']) : null;
+		$server_ca = isset($server_cert['caref']) ? str_replace("\n\n", "\n", str_replace("\r", "", ca_chain($server_cert))) : null;
```

Comment:
The `ca_chain()` function adds an empty line after each cert. I've adopted the double-`str_replace()` from the [CaptivePortal](https://github.com/opnsense/core/blob/b83fed8cfcc655f07db931e3961e6a92b85318cc/src/opnsense/scripts/OPNsense/CaptivePortal/generate_certs.php#L59). Didn't want to change the function, since it's used in the web gui config, too.